lib/Doctrine/DBAL/Event/SchemaAlterTableAddColumnEventArgs.php

```diff
@@ -57,17 +58,15 @@ public function getPlatform()
     }
 
     /**
+     * Passing multiple SQL statements as an array is deprecated. Pass each statement as an individual argument instead.
+     *
      * @param string|string[] $sql
```

Comment:
What's our policy about documenting variadicity or extra arguments achieved with `func_get_args()`? Should this be amended to reflect what @Ocramius described in https://github.com/doctrine/dbal/pull/3708/files#r341326501 ?

Reply:
Don't think there's a way to document it properly other than declaring it explicitly as parameter

Reply:
Not sure I'm understanding… I'm proposing to use `@param string|string[]|...string $sql`. Is that understood by SA tools/ IDEs?

Reply:
Answering my own question: phpstan does not like it. https://phpstan.org/r/52013d4d-e7c9-4c70-8b21-f28255ed96bd

Reply:
I was thinking this was going to be an issue, but apparently you can't trust the playground for that kind of thing: https://github.com/phpstan/phpstan/issues/1881#issuecomment-462816454